jobs/FunctionTest/cleanup.sh

```diff
@@ -58,6 +58,63 @@ cleanupENVProcess() {
   fi
 }
 
+
+
+clean_up_docker_image(){
+    # parameter : the images keyword to be delete
+    keyword=$1
+    images=`docker images | grep ${keyword} | awk '{print $3}' | sort | uniq`
+    if [ -n "$images" ]; then
+        docker rmi -f $images
+    fi
+}
+
+clean_running_containers() {
+    local containers=$(docker ps -a -q)
+    if [ "$containers" != "" ]; then
+        echo "Clean Up containers : " ${containers}
+        docker stop ${containers}
+        docker rm  ${containers}
+    fi
+}
+
+cleanupDocker(){
+  # Clean UP. (was in Jenkins job post-build, avoid failure impacts build status.)
+  set +e
+  set -x
+  echo "Show local docker images"
+  docker ps
+  docker images
+  echo "Stop & rm all docker running containers " 
+  clean_running_containers
+  echo "Chown rackhd/files volume on hosts"
+  echo $SUDO_PASSWORD |sudo -S chown -R $USER:$USER $WORKSPACE/RackHD 
+  echo "Clean Up all docker images in local repo"
+  clean_up_docker_image none
+  # clean images by order, on-core should be last one because others depends on it
+  clean_up_docker_image on-taskgraph
+  clean_up_docker_image on-http
+  clean_up_docker_image on-tftp
+  clean_up_docker_image on-dhcp-proxy
+  clean_up_docker_image on-syslog
+  clean_up_docker_image on-tasks
+  clean_up_docker_image files
+  clean_up_docker_image isc-dhcp-server
+  clean_up_docker_image on-wss
+  clean_up_docker_image on-statsd
+  clean_up_docker_image on-core
+  clean_up_docker_image rackhd
+
+
+}
+
+restart3rdServices(){
```

Comment:
when docker compose-up, mongo and rabbitmq docker  image will be up too.
Their containers  can't run if mongo  and rabbitmq services in host are still running.

Why not just use host services?
1. more aligned with user stories.
2. mongo need to be cleaned every time.